datasets/us/ofac/enforcement_actions/us_ofac_enforcement_actions.yml

```diff
@@ -0,0 +1,64 @@
+title: US OFAC Enforcement Actions
+entry_point: crawler.py
+prefix: ofac-ea
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
```

Comment:
fogive my copy pasta. I've since fixed it in cftc.  It's now load_statements: true
https://github.com/opensanctions/opensanctions/blame/main/datasets/us/cftc_enforcement_actions/us_cftc_enforcement_actions.yml#L3